* <pipping@mi.fu-berlin.de> fix building library to be proper Darwin style

--- configure.ac
+++ configure.ac
@@ -171,10 +171,10 @@
     ;;
   darwin*)
     SHLIBFORLINK=libnettle.dylib
-    SHLIBSONAME='$(SHLIBFORLINK).$(SHLIBMAJOR)'
-    SHLIBFILE='$(SHLIBSONAME).$(SHLIBMINOR)'
-    SHLIBLINK='$(CC) -dynamiclib $(LDFLAGS)'
-    # SHLIBLIBS=' -lgmp'
+    SHLIBSONAME='libnettle.$(SHLIBMAJOR).dylib'
+    SHLIBFILE='libnettle.$(SHLIBMAJOR).$(SHLIBMINOR).dylib'
+    SHLIBLINK='$(CC) -dynamiclib -install_name ${exec_prefix}/lib/$(SHLIBSONAME) $(LDFLAGS)'
+    SHLIBLIBS='$(LIBS)'
     ;;
   *)
     SHLIBFORLINK=libnettle.so
